Gemfile

```diff
@@ -4,38 +4,15 @@ source 'https://rubygems.org'
 ruby "2.4.4"
 git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
-plugin 'bootboot', '~> 0.1.1' unless Bundler.settings[:frozen]
```

Comment:
Just flagging that dualboot worked fine for me. I agree that it doesn't play well with Dependabot, manageable though. I haven't worked heaps with it yet, so can't really give a valid view on this.